src/utils.js

```diff
@@ -1477,6 +1477,11 @@ const bundleOrders = (
     for (let i = 0; i < ordersDetails.length; i++) {
         const orderDetails = ordersDetails[i];
         const orderStruct = JSON.parse(ordersDetails[i].orderJSONString);
+        // exchange the "handleIo" to "handleIO" in case "handleIO" is not present in order json
+        if (orderStruct.handleIO === undefined) {
```

Comment:
there's no test for the `if` condition

also, shouldn't we use something like lodash to normalize emptiness?

Reply:
dont think we need lodash, because it is only a check for an undefined key in the object